backend-go/internal/jwt/generator_test.go

```
@@ -0,0 +1,57 @@
+		RefreshTokenSecret:         "refresh-secret",
+		AccessTokenExpirationTime:  15 * time.Minute,
+		RefreshTokenExpirationTime: 7 * 24 * time.Hour,
+	}
+
+	gen := NewGenerator(cfg)
+
+	t.Run("validate access token test", func(t *testing.T) {
+		token, err := gen.GenerateAccessToken(dummyUserID, dummyNickname)
+		assert.NoError(t, err)
+
+		parsedToken, err := jwt.ParseWithClaims(token, &Payload{}, func(token *jwt.Token) (interface{}, error) {
```

golangci-lint (1.62.2)

[warning] unused-parameter: parameter 'token' seems to be unused, consider removing or renaming it as _

(revive)